README.md

```diff
@@ -55,6 +59,9 @@ Each service uses its own SQLite database. To access each database, simply open
 - maubot: `maubot/maubot.db`
 - go-neb: `go-neb/go-neb.db`
 
+### PostgreSQL database files
+PostgreSQL database is stored as files under `postgres/storage/`. Currently only `slack-bridge` uses PostgreSQL to store data.
```

Comment:
Knowing the location might be handy in case somebody wants an easy way to back up the Postgres data files. I will add the instructions for connecting to the database as well, good idea.